src/mlpack/bindings/python/print_input_processing.hpp

```diff
@@ -31,6 +31,11 @@ void PrintInputProcessing(
     const typename boost::disable_if<std::is_same<T,
         std::tuple<data::DatasetInfo, arma::mat>>>::type* = 0)
 {
+  // The copy_all_inputs parameter must be handled first, and so is outside the
```

Comment:
> and so is outside the

Is this correct?

Reply:
Yeah, I guess it could also be written `and therefore is outside the`; I went ahead and changed it for clarity.